examples/transformerconfigs/README.md

```diff
@@ -131,7 +131,7 @@ kind: Kustomization
 namespace: example
 
 resources:
-- resources.yaml
+  - resources.yaml
```

Comment:
Could you change every example to unify space less indent style like docs?
ref: https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/resource/

Reply:
+1 to this, agree with @koba1t that it would look better